R/print.R

```diff
@@ -71,10 +71,8 @@ print.gsplot <- function(x, ...){
     box()
   }
 
-  draw_legend(x)
-  draw_axis(x)
-
-  # par(defaultPar)
+  draw_legend(views)
+  draw_axis(views)
   
 }
 
```

Comment:
see this line: https://github.com/USGS-R/gsplot/pull/225/files#diff-832bc7f54d2e9d5191b3bdcef9979848R20

I think that is why